src/main/scala/reporter/RemoraDatadogReporter.scala

```diff
@@ -0,0 +1,41 @@
+package reporter
+
+import java.util.concurrent.TimeUnit
+
+import com.codahale.metrics.{Metric, MetricFilter, MetricRegistry}
+import config.DataDog
+import org.coursera.metrics.datadog.DatadogReporter
+import org.coursera.metrics.datadog.transport.UdpTransport
+
+/**
+  * Integration for DataDog
```

Comment:
Only complaint is that i don't think this comment adds value, the parameters and class names make sense to me.

Reply:
Sure, I will remove this